backend/wmg/data/transform.py

```diff
@@ -12,34 +12,11 @@
     PRIMARY_FILTER_DIMENSIONS_FILENAME,
 )
 from backend.wmg.data.tissue_mapper import TissueMapper
-from backend.wmg.data.utils import log_func_runtime
+from backend.wmg.data.utils import log_func_runtime, to_dict
 
 
 @log_func_runtime
-def get_cell_types_by_tissue(corpus_group: str) -> Dict:
```

Comment:
not needed anymore. cell types by tissue will be extracted from the cell counts cube.